db/snapshot-ctl.cc

```diff
@@ -16,6 +16,7 @@
 #include <seastar/coroutine/parallel_for_each.hh>
 #include "db/snapshot-ctl.hh"
 #include "replica/database.hh"
+#include "db/snapshot/backup_task.hh"
```

Comment:
nit, might want to move one line up, so it's closer to `#include "db/snapshot-ctl.hh"`.